packages/amplify-cli/src/extensions/amplify-helpers/remove-resource.js

```diff
@@ -56,11 +64,19 @@ async function removeResource(context, category, resourceName) {
     resourceName = answer.resource;
   }
 
-  console.log('');
+  context.print.info('');
+  let service = _.get(amplifyMeta, [category, resourceName, 'service']);
+  if (_.has(questionOptions, ['serviceDeletionInfo', service])) {
+    context.print.info(questionOptions.serviceDeletionInfo[service]);
+  }
+
   const resourceDir = path.normalize(path.join(pathManager.getBackendDirPath(), category, resourceName));
 
   return context.amplify.confirmPrompt
-    .run('Are you sure you want to delete the resource? This action deletes all files related to this resource from the backend directory.')
+    .run(
+      questionOptions.confirmPrompt ||
```

Comment:
Doesn't look like you're using questionOptions.confirmPrompt anywere. Did you mean to leave it in?

Reply:
Good catch! Fixed.